### PR TITLE
www-badges: Add badgeio styled SVG

### PR DESCRIPTION
**Pull request by jbenden**
This patch introduces a SVG template to match the look-and-feel of
badge.io badges.

In order to effectively mimic the look, some of the parameters
required tweaks, and the character font width calculation needed to be
corrected.

A running example of this change is located on the `README.md` of the [Aircrack-ng project](https://github.com/aircrack-ng/aircrack-ng/).

PS: I'm not sure if I need to add a file in `master/buildbot/newsfragments`. Nor am I sure if I need to add docs.

### CODE REVIEW

File: www/badges/buildbot_badges/__init__.py [hunk 1]
```diff
@@ -31,12 +31,12 @@ class Api:
     app = Klein()
 
     default = {  # note that these defaults are documented in configuration/www.rst
-        "left_pad": 5,
+        "left_pad": 0,
         "left_text": "Build Status",
         "left_color": "#555",
-        "right_pad": 5,
-        "border_radius": 5,
-        "style": "plastic",
```
Comment (tardyp) [inline www/badges/buildbot_badges/__init__.py]: why change the default? please document in www.rst example configuration with required default changes.

Comment (jbenden) [reply www/badges/buildbot_badges/__init__.py]: You're probably right, I shouldn't have changed the defaults. I've reverted those changes, while adding a documentation blurb with image.

Comment (jbenden) [reply www/badges/buildbot_badges/__init__.py]: PS: The font style change is really required, as all templates have incorrect character widths calculated; due to the mismatch in font weights used (bold for calculation and normal weight for display.)

**Comment by jbenden** [comment]
I'm not sure how to interpret the remaining CI failures. Please advise.

Thanks,
-Joe

**Comment by p12tic** [comment]
@jocelynj Looks great. Could you add a newsfragment describing this PR? See master/buildbot/newsfragments directory.

**Pull request merged**